src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,100 @@
+@import "../utils/variables";
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $product-card-width;
+  padding: 0 16px;
+  box-sizing: border-box;
+
+  &__image {
+    padding: 0 3px;
+    width: 160px;
+    height: 134px;
+    padding-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-navy-black;
+    margin-bottom: 4px;
+    margin-top: 0;
```

Comment:
Instead of reseting margins on each element you can create reset file where you will do it or you can use for example mayer reset: https://meyerweb.com/eric/tools/css/reset/

Reply:
I created separate [file ](https://github.com/AmadeuszAndroid/layout_catalog/blob/develop/src/styles/reset.scss)to reset h2 and p margins.